jsonnetfile.json

```diff
@@ -44,11 +44,11 @@
       "name": "observatorium",
       "source": {
         "git": {
-          "remote": "https://github.com/observatorium/observatorium",
```

Comment:
Shouldn't this be the other way around?

Reply:
It's still in a PR, that hasn't merged. I will update it before we merge this.